src/functionalBased/components/TodoContainer.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable */
+import React, { useState, useEffect } from "react";
+import { Route, Switch } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid";
+import About from "../pages/About";
+import NotMatch from "../pages/NotMatch";
+import Header from "./Header";
+import Navbar from "./Navbar";
+import InputTodo from "./InputTodo";
+import TodosList from "./TodosList";
+
+const TodoContainer = () => {
+  const [todos, setTodos] = useState(getInitialTodos());
+
+  const handleChange = (id) => {
+    setTodos((prevState) =>
+      prevState.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      })
+    );
+  };
+
+  const delTodo = (id) => {
+    setTodos([...todos.filter((todo) => todo.id !== id)]);
+  };
+
+  const addTodoItem = (title) => {
+    const newTodo = {
+      id: uuidv4(),
+      title,
+      completed: false,
+    };
+    setTodos([...todos, newTodo]);
+  };
+
+  const setUpdate = (updatedTitle, id) => {
+    setTodos(
+      todos.map((todo) => {
+        if (todo.id === id) {
+          todo.title = updatedTitle;
+        }
+        return todo;
+      })
+    );
+  };
+
+  function getInitialTodos() {
+    // getting stored items
+    const temp = localStorage.getItem("todos");
+    const savedTodos = JSON.parse(temp);
+    return savedTodos || [];
+  }
+
+  useEffect(() => {
+    // storing todos items
+    const temp = JSON.stringify(todos);
+    localStorage.setItem("todos", temp);
+  }, [todos]);
+
+  return (
+    <>
+      <Navbar />
+      <switch>
+        <Route exact path="/">
+          <div className="container">
+            <div className="inner">
+              <Header />
+              <InputTodo addTodoProps={addTodoItem} />
+              <TodosList
+                todos={todos}
+                handleChangeProps={handleChange}
+                deleteTodoProps={delTodo}
+                setUpdate={setUpdate}
+              />
+            </div>
+          </div>
+        </Route>
+        <Route path="/about">
+          <About />
+        </Route>{" "}
+        <Route path="*">
+          <NotMatch />
+        </Route>
+      </switch>
```

Comment:
You did a really great job implementing the path of pages but there are a few mistakes. Kindly fix them because the NotMatch page is showing on the home page.